configsamples/bgpadvertisement_with_node_selector.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: metallb.io/v1beta1
+kind: BGPAdvertisement
+metadata:
+  name: adv
```

Comment:
for consistency in the samples, I think we should keep the BGPAdvertisement names the same, as the first one: "advertisement"
(this applies to "test-advertisement1" above as well)